cookbook-react/src/recipes/radios/zecatDifapro/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import i18next from 'i18next';
+import clsx from 'clsx';
+
+import styles from './styles.module.scss';
+import ValidIcon from './assets/ic_valid.svg';
+import InvalidIcon from './assets/ic_invalid.svg';
+
+interface Props {
+  title: string;
+  info: string;
+  extraInfo: string;
+  disabled?: boolean;
+  name: string;
+  id: string;
+  inputStyle?: 'normal' | 'custom';
+  type?: 'radio' | 'checkbox';
+  checked: boolean;
+  onChange?: (event: React.FormEvent<HTMLInputElement>) => void;
+  wrapperClassName?: string;
+  titleClassName?: string;
+  infoClassName?: string;
+  extraInfoClassName?: string;
+}
+
+function ZecatDifapro({
```

Comment:
Add a comment above the first prop clarifying that these default values should be deleted in a project